src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
@nrk Many thanks for the info! I'd be great if you could confirm that this test would catch regressions: https://github.com/reactphp/event-loop/compare/master...jsor:ext-event-remove-error-suppression#diff-d6fed219f7ad523ad23fd345800fbf12